include/btllib/phred_nthash.hpp

```diff
@@ -0,0 +1,127 @@
+#ifndef BTLLIB_PHRED_NTHASH_HPP
+#define BTLLIB_PHRED_NTHASH_HPP
+
+#include "btllib/nthash.hpp"
+#include "btllib/util.hpp"
+
+#include <string>
+#include <vector>
+
+namespace btllib {
+/**
+ * NtHash with Phred score filtering.
+ */
+class PhredNtHash : private NtHash

```

Comment:
An alternative would be to use `: public NtHash` and hide unused methods with `using`, e.g.

```
private:
  using NtHash::peek;
```

This way, there'll be no need to reimplement `NtHash`'s public methods like `get_pos` etc.

Source: https://www.learncpp.com/cpp-tutorial/hiding-inherited-functionality/